source/data_mining/embedTweets.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Reads files of [[tweet,date]] Applies Universal Sentence Encoder to to the
+tweet to produce a third column with a 512 component embedding.  Writes a json
+output file containing this three column data.
+
+June, 2019
+@author: Joshua Rubin
+"""
+
+import os
+import json
+import pandas as pd
+import tensorflow as tf
+import tensorflow_hub as hub
+
+# Retrieve Universal Eentence Sncoder.
+module_url = "https://tfhub.dev/google/universal-sentence-encoder/2"
+embed = hub.Module(module_url)
+
+def embedTweets(inputDirectory, outputDirectory): 
```

Comment:
Doc string. Here is one style: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html, At the very least, what does this function do and what are the inputs/outputs (with type)

Reply:
Done.  And great reference.